tests/testthat/test-CombinationCohorts.R

```diff
@@ -0,0 +1,66 @@
+test_that("combination cohort generation", {
```

Comment:
This is a single, simple test, but it does perform some complicated operations such as loading the cohort table via `generateCohort` but the SQL passed in just inserts verbatim cohort records. This is so that we can pre-set specific overlapping date ranges so that we can confirm that the multiple cohorts got collapsed down into the expected result.